resources/variants/ultimaker3_bb0.8.inst.cfg

```diff
@@ -39,7 +40,8 @@ material_print_temperature = =default_material_print_temperature + 10
 material_standby_temperature = 100
 multiple_mesh_overlap = 0
 prime_tower_enable = False
-prime_tower_purge_volume = 1
+prime_tower_purge_volume = 2
+prime_tower_wall_thickness = 2.2
```

Comment:
Maybe something like `=prime_tower_line_thickness * 3`?

Reply:
Unfortunately Cura does not handle this well, as it will do weird short line pieces instead of three walls with that setting (bugged thin walls). =prime_tower_line_width * 3 + 0.1  would work though.

Reply:
I'll keep it this way for now, but maybe you can try again using the Try Multiple Line Thicknesses setting. It's an ongoing test to fix those broken walls.